src/Razor/test/Microsoft.AspNetCore.Razor.LanguageServer.Test/Semantic/RazorSemanticTokenInfoServiceTest.cs

```diff
@@ -635,14 +711,16 @@ public async Task GetSemanticTokens_Razor_MultiLineCommentWithBlankLines()
                 @* kdl
 
                 skd
-                    
```

Comment:
Why this change? Did the test fail, or was VS just trying to be helpful (and if so, we should file a bug on Roslyn)?

This looks like a valid test case.

Reply:
I'm guessing it's fine to remove the unnecessary whitespace. with or without it the test passes without a problem

Reply:
Given that its part of the test input, I wouldn't call it unnecessary whitespace. It's possible that its validating behaviour that previously caused bugs, so updating the test input could mean we inadvertently reintroduce a bug without knowing.

To be very clear here: The way raw string literals work in C#, it means the first 16 spaces on the line are not test input, but the 4 spaces after column 16 are. It's surprising at first, but essentially the compiler only considers things after the column that the trailing `"""` starts at as part of the string.

Reply:
Sure I'll bring the whitespace back